install/upgrade/versions/1.9.4.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Hestia Control Panel upgrade script for target version 1.9.3
+
+#######################################################################################
+#######                      Place additional commands below.                   #######
+#######################################################################################
+####### upgrade_config_set_value only accepts true or false.                    #######
+#######                                                                         #######
+####### Pass through information to the end user in case of a issue or problem  #######
+#######                                                                         #######
+####### Use add_upgrade_message "My message here" to include a message          #######
+####### in the upgrade notification email. Example:                             #######
+#######                                                                         #######
+####### add_upgrade_message "My message here"                                   #######
+#######                                                                         #######
+####### You can use \n within the string to create new lines.                   #######
+#######################################################################################
+
+upgrade_config_set_value 'UPGRADE_UPDATE_WEB_TEMPLATES' 'false'
+upgrade_config_set_value 'UPGRADE_UPDATE_DNS_TEMPLATES' 'false'
+upgrade_config_set_value 'UPGRADE_UPDATE_MAIL_TEMPLATES' 'false'
+upgrade_config_set_value 'UPGRADE_REBUILD_USERS' 'no'
+upgrade_config_set_value 'UPGRADE_UPDATE_FILEMANAGER_CONFIG' 'true'
+
+if grep -q "internal-sftp-server" /etc/ssh/sshd_config; then
+	# add a line before the line that contains "internal-sftp-server"
+	sed -i '/Subsystem sftp internal-sftp-.*/i Subsystem sftp internal-sftp' /etc/ssh/sshd_config
+fi
+
+# When jail bash is active
+if grep -q "Subsystem sftp /usr/lib/sftp-server-" /etc/ssh/sshd_config; then
+	sed -i '/Subsystem sftp \/usr\/lib\/sftp-server-*./Subsystem sftp \/usr\/lib\/sftp-server' /etc/ssh/sshd_config
```

Comment:
Question: Might the `*.` be turned around here? Or are you trying to match multiple `-` ? I might be reading the regex wrong here.

Reply:
You are right